src/jdk.compiler/share/classes/com/sun/tools/javac/tree/Pretty.java

```diff
@@ -132,6 +132,12 @@ public void print(Object s) throws IOException {
         out.write(Convert.escapeUnicode(s.toString()));
     }
 
+    /** Print character.
+     */
+    private void print(char c) throws IOException {
+        out.write(c);
+    }
```

Comment:
This bypasses the `Convert.escapeUnicode` code on line 132.   That's not bad, but should be clearly documented ... in that this is a special method for known ASCII characters, and not a general purpose method that could be used on the characters of a string

Reply:
Adjusted the comment in that regards.